datahub-web-react/src/graphql-mock/fixtures/browseDataset/datasetBrowseKafka.ts

```diff
@@ -0,0 +1,207 @@
+                    __typename: 'BrowseResultMetadata',
+                },
+                __typename: 'BrowseResults',
+            },
+        },
+    };
+};
+
+export const datasetBrowseProdKafkaTwineCorollary = ({ start }): GetBrowseResults => {
+    return {
+        data: {
+            browse: {
+                entities: datasets.slice(2, 3),
+                start,
+                count: 1,
+                total: 1,
+                metadata: {
+                    path: ['prod', 'kafka', 'twine', 'corollary'],
+                    groups: [],
+                    totalNumEntities: 1,
+                    __typename: 'BrowseResultMetadata',
+                },
+                __typename: 'BrowseResults',
+            },
+        },
+    };
+};
+
+export const datasetBrowseProdKafkaTwineHitch = ({ start }): GetBrowseResults => {
+    return {
+        data: {
+            browse: {
+                entities: datasets.slice(3, 4),
+                start,
+                count: 1,
+                total: 1,
+                metadata: {
+                    path: ['prod', 'kafka', 'twine', 'hitch'],
+                    groups: [],
+                    totalNumEntities: 1,
+                    __typename: 'BrowseResultMetadata',
+                },
+                __typename: 'BrowseResults',
+            },
+        },
+    };
+};
+
+export const datasetBrowseProdKafkaWah: GetBrowseResults = {
+    data: {
+        browse: {
+            entities: [],
+            start: 0,
+            count: 0,
+            total: 0,
+            metadata: {
+                path: ['prod', 'kafka', 'wah'],
+                groups: [
+                    { name: 'agnes', count: 1, __typename: 'BrowseResultGroup' },
+                    { name: 'corollary', count: 2, __typename: 'BrowseResultGroup' },
+                ],
+                totalNumEntities: 3,
+                __typename: 'BrowseResultMetadata',
+            },
+            __typename: 'BrowseResults',
+        },
+    },
+};
+
+export const datasetBrowseProdKafkaWahAgnes = ({ start }): GetBrowseResults => {
+    return {
+        data: {
+            browse: {
+                entities: datasets.slice(4, 5),
+                start,
+                count: 1,
+                total: 1,
+                metadata: {
+                    path: ['prod', 'kafka', 'wah', 'agnes'],
+                    groups: [],
+                    totalNumEntities: 1,
+                    __typename: 'BrowseResultMetadata',
+                },
+                __typename: 'BrowseResults',
+            },
+        },
+    };
+};
+
+export const datasetBrowseProdKafkaWahCorollary = ({ start }): GetBrowseResults => {
+    return {
+        data: {
+            browse: {
+                entities: datasets.slice(5, 7),
+                start,
+                count: 2,
+                total: 2,
+                metadata: {
+                    path: ['prod', 'kafka', 'wah', 'corollary'],
```

Comment:
@wan54  can you make this sample data a little more realistic? I worry someone might boot up this mock server and be confused when they see words like this.

Reply:
I know http://demo.datahubproject.io/ had words like this in certain instances but we removed them due to them being confusing.